### ISSUE TITLE
Update with the update button within Metabase

### ISSUE THREAD

**Issue by biva** [YunoHost-Apps/metabase_ynh#19]
Hello,

Thank you very much for this package! We really enjoy using it. I see there's an update button within Metabase (mydomain.com/admin/settings/updates), proposing an update from 0.47.0 to 0.47.10.

Is there a risk to update Metabase from here, instead of waiting for a release of the package on Yunohost?

Thank you again

**Comment by stepcellwolf** [comment]
Hi, @biva did you managed to find away to update your metabase?

**Comment by biva** [comment]
Nope, just using the normal update coming from the Yunohost package

**Comment by Enthouane** [comment]
I'm also wondering if i can update through the metabase webadmin or if a package update is coming...
By the way thanks a lot for packaging this great app !
